### DISCUSSION TITLE
Nuxt 3 compatibility

### DISCUSSION THREAD

**Discussion by mirkopoloni** [frinyvonnick/node-html-to-image#183] [Q&A] [answered]
Hello everybody, I tried to implement the basic example of this library with a Nuxt 3 project but I can't find a way to make it work.
This is the error that I am seeing: `Class extends value undefined is not a constructor or null at node_modules/puppeteer-cluster/dist/Cluster.js`
It could have something to do with how Nuxt handles ESM Modules or maybe Vite...I am not sure.
Has anybody managed to use the library with a Nuxt 3 project?
Thanks.

**Comment by mirkopoloni** [comment, marked as answer]
I managed to make it work, the solution was an obvious one: just make sure that you are using the library only on server side and not on client side.
I'll leave it here just in case someone will face the same problem.

**Reply by frinyvonnick** [reply]
Hello @mirkopoloni 👋 Thank you for taking the time to post the answer 🙏

**Reply by mirkopoloni** [reply]
Hey @frinyvonnick thank you for the great library!
The Nuxt's error somehow pushed me in the wrong direction. I have definitely spent too many hours debugging what actually was a basic error on my side 🙈 
Thanks again.
